src/components/Select/Select.tsx

```diff
@@ -109,39 +120,58 @@ const Select = React.forwardRef<
       "NYPL Reservoir Select: This component's required `id` prop was not passed."
     );
   }
+
+  useEffect(() => {
+    if (labelPosition === LabelPositions.Inline) {
+      const width = labelRef?.current?.clientWidth + 8;
+      setLabelWidth(width);
+    }
+  }, [labelRef.current]);
```

Comment:
There's a linting issue here with `labelPosition` missing here.

Reply:
I added labelPosition and then I got a linting error about labelRef.current, so I adjusted again. I think it's okay now.